src/ragas/metrics/_faithfulness.py

```diff
@@ -42,6 +42,9 @@ class Statements(BaseModel):
 class StatementsAnswers(RootModel):
     root: t.List[Statements]
 
+    def dicts(self):
+        return self.model_dump()
+
```

Comment:
Nice catch! the proper fix would be to replace the lines that use this function with `self.model_dump()` to stay consistant with the rest of the codebase.

could you rework the PR so that it changes those lines?

Reply:
I just reworked it! I noticed that the dicts method was still used in other parts of the code base (for the faithfulness metric and in noise sensitivity) so I also replaced it there